### DISCUSSION TITLE
How to make contributions?

### DISCUSSION THREAD

**Discussion by roeest** [nautilus/gateway#178] [Q&A] [answered]
Hi,

I encountered an issue when using a more recent version of gqlgen (0.17.3) along with nautilus, i have a pr ready however i dont have permissions to create a branch and a pr consequently. What would be the best way to get it pushed?

**Comment by JohnStarich** [comment, marked as answer]
Hey @roeest, thanks for reaching out! You can fork this repo and open a PR from there. Here's a GitHub support article that walks you through it after you've created the fork: https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork
